### PR TITLE
Fix the bug that Chinese characters are garbled in ResponseBody of ZuulBlockFallbackProvider

### LINKED ISSUE #733: ZuulBlockFallbackProvider ResponseBody Chinese garbled
## Issue Description

Type: *bug report* 

### Describe what happened (or what feature you want)

ZuulBlockFallbackProvider ResponseBody 中文乱码

### Describe what you expected to happen

支持中文message

### How to reproduce it (as minimally and precisely as possible)

1. 增加一行 ctx.getResponse().setContentType("application/json; charset=utf-8");即可
image

### Tell us your environment

sentinel-zuul-adapter 1.6.0版本

### PR DESCRIPTION
**Pull request by yinjihuan**
### Describe what this PR does / why we need it

Fix the bug that Chinese characters are garbled in ResponseBody of ZuulBlockFallbackProvider.

### Does this pull request fix one issue?

Fixes #733 

### Describe how you did it

Add a line `ctx.getResponse().setContentType("application/json; charset=utf-8");`

### Describe how to verify it

Custom ZuulBlockFallbackProvider, Set the message field to Chinese，Trigger current limiting，Message field Chinese display is normal

### Special notes for reviews

NO

### CODE REVIEW

File: sentinel-demo/sentinel-demo-zuul-gateway/src/main/java/com/alibaba/csp/sentinel/demo/zuul/gateway/MyBlockFallbackProvider.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+	@Override
+	public String getRoute() {
+		return "*";
+	}
+
+	@Override
+	public BlockResponse fallbackResponse(String route, Throwable cause) {
+		if (cause instanceof BlockException) {
+			return new BlockResponse(429, "限流异常", route);
```
Comment (sczyh30) [inline sentinel-demo/sentinel-demo-zuul-gateway/src/main/java/com/alibaba/csp/sentinel/demo/zuul/gateway/MyBlockFallbackProvider.java]: Please do not add Chinese directly in the demo. You can just leave the code in the PR description.

Comment (yinjihuan) [reply sentinel-demo/sentinel-demo-zuul-gateway/src/main/java/com/alibaba/csp/sentinel/demo/zuul/gateway/MyBlockFallbackProvider.java]: I see the requirement for additional test cases

Comment (yinjihuan) [reply sentinel-demo/sentinel-demo-zuul-gateway/src/main/java/com/alibaba/csp/sentinel/demo/zuul/gateway/MyBlockFallbackProvider.java]: What to do next

Comment (sczyh30) [reply sentinel-demo/sentinel-demo-zuul-gateway/src/main/java/com/alibaba/csp/sentinel/demo/zuul/gateway/MyBlockFallbackProvider.java]: Yes, we often add unit test (not via the demo). You can just keep your change for the `SentinelZuulPreFilter` class and describe how to verify it in the PR description.

---

通常我们要在模块中添加单元测试来验证，而不是在 demo 中修改然后观察。你可以去掉 demo 的更改，保留 `SentinelZuulPreFilter` 类的更改，然后在 PR 里面 edit 描述一下如何验证就好。

Comment (yinjihuan) [reply sentinel-demo/sentinel-demo-zuul-gateway/src/main/java/com/alibaba/csp/sentinel/demo/zuul/gateway/MyBlockFallbackProvider.java]: 好的，那是不是得重新提个PR

Comment (sczyh30) [reply sentinel-demo/sentinel-demo-zuul-gateway/src/main/java/com/alibaba/csp/sentinel/demo/zuul/gateway/MyBlockFallbackProvider.java]: 不用，直接在当前 PR 接着修改就好，PR 描述也可以编辑。

Comment (yinjihuan) [reply sentinel-demo/sentinel-demo-zuul-gateway/src/main/java/com/alibaba/csp/sentinel/demo/zuul/gateway/MyBlockFallbackProvider.java]: 那测试案例的代码，我本地删除，然后重新push就可以了对吧

Comment (sczyh30) [reply sentinel-demo/sentinel-demo-zuul-gateway/src/main/java/com/alibaba/csp/sentinel/demo/zuul/gateway/MyBlockFallbackProvider.java]: That's okay.

**Comment by sczyh30** [comment]
Thanks for contributing!

**Pull request merged**